guides/installation_guides/c++/mac/README.md

```diff
@@ -1,7 +1,35 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Cosmos Guide: Getting Ready to program in C++ in MAC OS
```

Comment:
Looks like your PRs are getting mixed; You have to use [topic branches](https://gist.github.com/vlandham/3b2b79c40bc7353ae95a).

Reply:
I have reviewed on your other PR for the egg drop puzzle.

Reply:
okayy thank you 👍